### DISCUSSION TITLE
When will \below be supported??

### DISCUSSION THREAD

**Discussion by sauravkhakurel** [KaTeX/KaTeX#3671] [General]
I was trying to render some equation with \below but could not do so, \above is listed in the supported functions page but could not find \below. Will \below ever be supported in katex? Thanks

**Comment by edemaine** [comment]
What is `\below`? Could you give an example of its usage, or a link where it's defined?

Is https://github.com/mathjax/MathJax/issues/2262 relevant? Perhaps `\below` is a MathJax-ism that translates to `\limits_` in LaTeX/KaTeX?
